test/test.js

```diff
@@ -3,95 +3,109 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inBrowser = typeof window !== 'undefined';
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = !inBrowser ? require('jsdom') : null;
+
+var doc = inBrowser ? document : jsdom.jsdom();
+var win = inBrowser ? window : doc.parentWindow;
+
 describe('get-window', function () {
 
   it('should work with the global Window object', function () {
-    var win = getWindow(window);
-    assert(win === window);
+    var result = getWindow(win);
+    assert(result === win);
   });
 
   it('should work with the global Document object', function () {
-    var win = getWindow(document);
-    assert(win === window);
+    var result = getWindow(doc);
+    assert(result === win);
   });
 
   it('should work with the `document.body` object', function () {
-    var win = getWindow(document.body);
-    assert(win === window);
+    var result = getWindow(doc.body);
+    assert(result === win);
   });
 
   it('should work with the a DOM element inside `document.body`', function () {
-    var win = getWindow(document.body.firstChild);
-    assert(win === window);
+    var p = doc.createElement('p')
+    doc.body.appendChild(p);
+    var result = getWindow(doc.body.firstChild);
+    assert(result === win);
+    doc.body.removeChild(p);
```

Comment:
@TooTallNate idempotent. You're right. So removing the child, afterwards, like in other specs.